### ISSUE TITLE
Search with Cree preverbs does not produce results

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/cree-intelligent-dictionary#161]
Searching e.g. with the Cree preverbs _ati-_, _poni-_, _nitawi-_, and _pe-_ do not produce results.

In contrast, search with English words corresponding to the above does result in list with some but not all of the preverbs above.

**Comment by Madoshakalaka** [comment]
This is fixed in my development branch (not on sapir yet).
ati-, poni-, pe- now all produce results. yet nitawi- does not. "nitaw" without "i" does. was that a typo?

**Comment by eddieantonio** [comment]
"nitawi-" is most definitely not a typo: https://altlab.ualberta.ca/itwewina/crk/eng/?lookup=nitawi-

**Comment by Madoshakalaka** [comment]
I found the cause: these two entries exist in the source

```
<e>
   <lg>
      <l pos="Ipc">nitaw</l>
      <lc>IPV</lc>
      <stem>md_stem[nr]</stem>
   </lg>
   <mg>
   <tg xml:lang="eng">
       <t pos="Ipc" sources="MD">Meaning, go and get, go and do or go and act. Also nitawi.</t>
   </tg>
   </mg>
</e>

<e>
   <lg>
      <l pos="Ipc">nitaw</l>
      <lc>IPV</lc>
      <stem>md_stem[nr]</stem>
   </lg>
   <mg>
   <tg xml:lang="eng">
       <t pos="Ipc" sources="MD">Meaning, go and get, go and do or go and act. Also nitawi.</t>
   </tg>
   </mg>
</e>

```

The pending dash is not properly dealt with in the current algorithm.

**Comment by Madoshakalaka** [comment]
I will come up with tests for these cases

**Comment by Madoshakalaka** [comment]
@aarppe need your advice what do we show as the part of speech for reduplication stems like "sa" "ka" "ma" "na". 

```
<e>
   <lg>
      <l pos="IPV">sa</l>
      <lc></lc>
      <stem>md_stem[nr]</stem>
   </lg>
   <mg>
   <tg xml:lang="eng">
       <t pos="IPV" sources="MD">Usually added to a verb to mean continuous action. [Reduplication]</t>
   </tg>
   </mg>
</e>
```

Since they all have `pos="IPV"` in their sources. I currently show them as "Preverbs" in my development branch

![Screenshot from 2019-11-27 15-24-54](https://user-images.githubusercontent.com/44753941/69763329-182b2200-112a-11ea-8fed-69926f056257.png)

**Comment by aarppe** [comment]
The short answer is that, for now, calling the reduplicative elements preverbs is the simplest solution, as the do look like some grammatical preverbs, even though strictly speaker they are not preverbs (the rough meanings for the Ca- and Câh- patterns are the same irrespective of what the initial Consonant is).

I don't think Arok's dictionary lists most reduplicative elements for the most part, but the Maskwacîs folks thought they were relevant. However, in MD the preverbs don't have the hyphen, but since they are not independent words, in SRO they should be written with a hyphen to indicate that they always need to co-occur with a stem or the like.

(Note for later reference: in syllabics, preverbs would be written without a hyphen, so separated with a space from the stems, but I don't know if this applies for reduplicative elements or whether they should be attached to the stem).

**Comment by eddieantonio** [comment]
Fixed as of 24b6a4fccf79cb7cb1bbef32f7d4776a86f384d8
